stack.py

```diff
@@ -1,33 +1,20 @@
-class Node:
-    def __init__(self, value):
-        self.value = value
-        self.next = None
+from linked_list import LinkedList
 
-class Stack:
+class Stack(LinkedList):
     def __init__(self, iterable=None):
         self.top = None
+        self.head = None
         self._size = 0
 
         if iterable is not None:
             for item in iterable:
                 self.push(item)
```

Comment:
Here is a Jumping off point.

```suggestion
    def __init__(self, iterable=None):
        self._storage = LinkedList(iterable)
```